frontend/src/types/api/alerts/create.ts

```diff
@@ -46,3 +49,97 @@ export const alertDefaults: AlertDef = {
 	},
 	evalWindow: defaultEvalWindow,
 };
+
+export const logAlertDefaults: AlertDef = {
```

Comment:
why we are polluting default here ?

Reply:
not sure what you mean. The log and trace defaults are different than metric default. I created separate vars to accommodate upcoming changes like log specific labels, etc. 
is there an alternative approach we could take?

Reply:
I moved the defaults to a separate file under container/createAlertRule. as it is initial variable default value.